source/Sound.Flac/Decoder.cs

```diff
@@ -179,7 +179,7 @@ internal static class Decoder {
 					 * block may be shorter than the stream blocksize; its starting sample number will be calculated as the frame number 
 					 * times the previous frame's blocksize, or zero if it is the first frame). 
 					 */
-					ulong sampleOrFrameNumber = (ulong)reader.ReadUTF8EncodedInteger();
+					reader.ReadUTF8EncodedInteger(); // ulong sampleOrFrameNumber = (ulong)
```

Comment:
Yup, but at least until we can fix it, we still have to read the value and discard it so that everything else is in the right place.